.travis.yml

```diff
@@ -60,6 +62,7 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+      - RUSTFLAGS="-D warnings" cargo package --verbose --allow-dirty
```

Comment:
We can avoid doing this on macOS and Windows, since realistically we'll only run this on Linux.

Reply:
Ok. I left `cargo package` in linux build only.